README.md

```diff
@@ -158,10 +164,15 @@ class HandlerJob extends Job
         // Raw JSON payload from SQS, if necessary
         var_dump($job->getRawBody());
     }
-}
 
+    public function getRawBody() {}
+
+    public function getJobId() {}
+}
```

Comment:
actually, I think we can just remove "extends Job" completely (and therefore the two extra methods). All Laravel needs from a job class is the "handle" method, nothing else matters. This is the reason for confusion I think